sftp-client.h

```diff
@@ -144,6 +144,21 @@ int do_upload(struct sftp_conn *, const char *, const char *, int, int, int);
 int upload_dir(struct sftp_conn *, const char *, const char *, int, int, int,
     int);
 
+/*
+ * Download a 'from_path' from the 'from' connection and upload it to
+ * to 'to' connection at 'to_path'.
+ */
+int do_crossload(struct sftp_conn *, struct sftp_conn *,
```

Comment:
you need to keep the argument names for the comment

Reply:
fixed